src/OmniSharp.Roslyn.CSharp/Services/Types/TypeLookup.cs

```diff
@@ -15,6 +15,14 @@ public class TypeLookupService : RequestHandler<TypeLookupRequest, TypeLookupRes
     {
         private readonly FormattingOptions _formattingOptions;
         private readonly OmnisharpWorkspace _workspace;
+        private static readonly SymbolDisplayFormat DefaultFormat = SymbolDisplayFormat.FullyQualifiedFormat.WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted);
+        private static readonly SymbolDisplayFormat SimpleFormat = new SymbolDisplayFormat(
+                globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Omitted,
+                typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypes,
+                genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
+                miscellaneousOptions:
+                    SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
+                    SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
```

Comment:
However `MinimallyQualifiedFormat` skips namespace / containing type information. For example calling `symbol.ToMinimalDisplay...` on `Foo.Bar.Xyz` returns just `Xyz`, is that the intention? Because OmniSharp has shown fully qualified info